### PR TITLE
feat(connector archetype) add detailed example

### PR DESCRIPTION
**Pull request by alachambre**
* Connector in Groovy using retrofit to perform http call on
https://swapi.dev/
* Example is detailed and explained step by step, to guide users in
their custom connector development journey.

### CODE REVIEW

**Comment by uguy** [comment]
LGTM :)
Just asking myself if it would be better to _"include"_ raw part of code from the github project itself instead of duplicating it in this doc ?

**Review by uguy — COMMENTED** [review]
LGTM :)
Just asking myself if it would be better to "include" raw part of code from the github project itself instead of duplicating it in this doc ?
In any case, it would be good to have this code in a repo and a link pointing to it.

**Comment by alachambre** [comment]
I don't know if this github project will be kept for long. 
If we want to do such a thing then we must deplace it in the Bonitasoft organization, etc. 
Usually our documentation pages embed their own piece of code.

**Comment by uguy** [comment]
Ok for me then !

File: md/connector-archetype.md [hunk 1]
```diff
@@ -265,3 +265,504 @@ To build the connector project, type the following command at the root of the pr
...
+}
+
+/**
+ * Connector unit test - no internet required
+ */
+def should_fetch_person() {
+    given: 'A person name'
+    def name = 'Luke'
+    and: 'A related person JSON response'
+    def body = "{\"count\":1,\"next\":null,\"previous\":null,\"results\":[{\"name\":\"$name Skywalker\",\"height\":\"172\",\"mass\":\"77\",\"hair_color\":\"blond\",\"skin_color\":\"fair\",\"eye_color\":\"blue\",\"birth_year\":\"19BBY\",\"gender\":\"male\",\"homeworld\":\"http://swapi.dev/api/planets/1/\",\"films\":[\"http://swapi.dev/api/films/1/\",\"http://swapi.dev/api/films/2/\",\"http://swapi.dev/api/films/3/\",\"http://swapi.dev/api/films/6/\"],\"species\":[],\"vehicles\":[\"http://swapi.dev/api/vehicles/14/\",\"http://swapi.dev/api/vehicles/30/\"],\"starships\":[\"http://swapi.dev/api/starships/12/\",\"http://swapi.dev/api/starships/22/\"],\"created\":\"2014-12-09T13:50:51.644000Z\",\"edited\":\"2014-12-20T21:17:56.891000Z\",\"url\":\"http://swapi.dev/api/people/1/\"}]}"
```
Comment (rbioteau) [inline md/connector-archetype.md]: use groovy multiline string for readability

File: md/connector-archetype.md [hunk 2]
```diff
@@ -265,3 +265,504 @@ To build the connector project, type the following command at the root of the pr
...
+* **bonitaVersion**: _[Your Bonita version]_ (ex: 7.10.4)
+* **className**: ConnectorStarWars
+* **language**: groovy
+* **wrapper**: true
+
+The generated project has the following skeleton:  
+
+![Connector project overview](images/connector-archetype-example/connector-project-overview.png)  
+
+Add the following properties and dependencies to the existing ones in the pom.xml:
```
Comment (rbioteau) [inline md/connector-archetype.md:300]: Explain what is retrofit and add a link to retrofit home page

**Review by rbioteau — COMMENTED** [review]
Great example
It could be nice to add test coverage configuration (jacoco maven plugin)

File: md/connector-archetype.md (hunk 2 above)
Comment (alachambre) [reply md/connector-archetype.md:300]: already done, first lines of the tuto. 
And a short explanation comes later.

Comment (rbioteau) [reply md/connector-archetype.md:300]: You do not explain what it does...

**Pull request merged**